apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-credit/footerCreditChoices.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable wpcalypso/import-docblock */
+/* global fullSiteEditing */
+/**
+ * WordPress dependencies
+ */
+import { Icon } from '@wordpress/components';
+
+export const { footerCreditOptions, defaultCreditOption } = fullSiteEditing;
+
+export const RenderedCreditChoice = ( { choice } ) => {
+	const selection = footerCreditOptions.find( ( { value } ) => value === choice );
+	if ( ! selection ) {
+		return null;
+	}
+
+	const { renderType, renderProps, label } = selection;
+	// Allows label to be overriden by renderProps if needed.
+	const props = { label, ...renderProps };
+	if ( 'icon' === renderType ) {
+		return <Icon { ...props } />;
+	}
+	return <span> { props.label } </span>;
+};
```

Comment:
File names should be in `kebab-case` instead of `snakeCase`.
I can see that we have several cases of snake cases in the FSE plugin, but I guess they slipped through the reviews. 🙂

Reply:
Thanks! I'll change this